index.html

```diff
@@ -1,117 +1,283 @@
-                </div>
+<body  >
+    <div id="nav-wrapper" >
+           <div class="navbar navbar-inverse " >
+        <div class="container">
+            <div class="navbar-header ">
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+                
+                <a class="navbar-brand" href="#"><img src="assets/img/design.png" width = "200" height = "70"></a>
             </div>
+            <div class="navbar-collapse collapse move-me">
+                <ul class="nav navbar-nav navbar-right">
+                    <li><a href="#home-sec">Home</a></li>
+                     <li><a href="#projects-sec">Potential Projects</a></li>
+                    <li><a href="#schedule-sec">Event Schedule</a></li>
+                     <li><a href="#rules-sec">ChariTech Rules</a></li>
+                     <li><a href="#contact-sec">Contact</a></li>
+                </ul>
+            </div>
+           
         </div>
     </div>
+        </div>
+   <!-- NAVBAR SECTION END-->
+     <section id="home-sec"> 
+           <div class="overlay"  >
+               <div class="container" >
+                <div class="row text-center">
+                    <div class="col-md-12">
+                      <h1 ><font color = "black">  </font></h1> 
+                         <h2>
+                         <font color = "black"> Hackers at Cambridge presents the hackathon <br> aimed at helping out charities. <br><br>
+                         Innovate, develop, have fun! 
+                         </font>
 
-<div class="container">
-      <div class="row">
-        <div class="col s12 m4">
-          <div class="card" id="schedule">
-            <div class="card-content">
-              <span class="card-title">Schedule</span>
-              <ul>
-                <li id="t1000">10:00 Arrival</li>
-                <li id="t1030">10:30 LT1 Presentations from our Charities begin!</li>
-                <li id ="t1200">12:00 Lunch</li>
-                <li id ="t1800">18:00 Dinner</li>
-                <li id ="t2000">20:00 Stop Hacking!</li>
-                <li id ="t2110">21:10 LT1 Prizes and wrap-up</li>
-              </ul>
+                        </h2>
+                        <div class="card-content">
+                          <iframe src="http://free.timeanddate.com/countdown/i5mmb2ti/n1234/cf12/cm0/cu4/ct0/cs1/ca0/co1/cr0/ss0/cac000/cpc000/pct/tcfff/fs100/szw576/szh243/tatTime%20left%20to%20ChariTech/tac000/tptTime%20since%20Event%20started%20in/tpc000/iso2017-03-17T10:00:00" allowTransparency="true" frameborder="0" width="354" height="87"></iframe>
```

Comment:
Was just a fast thing I could come up with, I see your point now that you mention it. Plus I also have to think what to do when the event actually starts, I may try to adapt the gamegig one, time permitting.